common/go-utils/src/inv.rs

```diff
@@ -23,6 +25,18 @@ pub struct Artifact {
     pub sha_checksum: String,
 }
 
+impl Hash for Artifact {
+    fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
+        self.sha_checksum.hash(state);
+    }
+}
+
+impl Display for Artifact {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{} ({} - {})", self.go_version, self.os, self.arch)
```

Comment:
Did you mean for the os/arch pair to be displayed as `linux - x86_64`? 

```suggestion
        write!(f, "{} ({}-{})", self.go_version, self.os, self.arch)
```

Reply:
This `impl` should have a unit test

Reply:
That is a prettier way to display it. I've committed your suggestion, and added a unit test for the display implementation https://github.com/heroku/buildpacks-go/pull/230/commits/663e53805fd07e36e1967b64d3cbc53b426a0f68